packages/start/createCompiler.js

```diff
@@ -22,6 +21,44 @@ function getMaxSeverityErrors(errors) {
   return errors.filter((e) => e.severity === maxSeverity);
 }
 
+function configureTypeChecker(config) {
+  if (Array.isArray(config)) return config.map(configureTypeChecker);
+
+  const { plugins = [], ...rest } = config;
+
+  const index = plugins.findIndex(
+    (p) => p.constructor.name === 'ForkTsCheckerWebpackPlugin',
+  );
+
+  if (index > -1 && !plugins[index].constructor.version)
+    throw new Error(
+      '@4c/start is only compatible with ForkTsCheckerWebpackPlugin v5 and above',
+    );
+
+  const options = index !== -1 ? plugins[index].options : {};
+
+  const plugin = new ForkTsCheckerWebpackPlugin({
```

Comment:
the way this structured now i can't monkey patch it so we need to just find and replace the plugin before it's applied